indico/modules/rb_new/event/fields.py

```diff
@@ -0,0 +1,62 @@
+# This file is part of Indico.
+# Copyright (C) 2002 - 2019 European Organization for Nuclear Research (CERN).
+#
+# Indico is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 3 of the
+# License, or (at your option) any later version.
+#
+# Indico is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Indico; if not, see <http://www.gnu.org/licenses/>.
+
+from __future__ import unicode_literals
+
+from wtforms.ext.sqlalchemy.fields import QuerySelectField
+
+from indico.util.i18n import _
+from indico.web.flask.util import url_for
+from indico.web.forms.widgets import SelectizeWidget
+
+
+class LinkedObjectField(QuerySelectField):
```

Comment:
QSF doesn't make much sense here - since we don't use the form for submission even a simple `Field` should do the job for rendering